### ISSUE TITLE
feature request: Multisite information for super admin

### ISSUE THREAD

**Issue by colomet** [bonny/WordPress-Simple-History#135]
To be able to see all the issues of the multisite in the super admin dashboard would help to control the network.

Maybe something like WP Security Audit Log does in their multisite view.

thanks

**Comment by khromov** [comment]
+1 this. We're trying to migrate from Stream to Simple History but not having a centralised dashboard of everything happening on the multisite is a no-go. 

If multisite is detected, everything should be stored in `ab_simple_history` table and additional columns for `network_id` and `site_id` should be added.

**Comment by bonny** [comment]
Not sure I understand what you're looking for. Is it a view in the network admin where you can see everything that happened anywhere on any of the sites (like someone created a post on site A and someone failed to login on site B) or do you want to see events that happen in the network admin (a site is created, a user is added to a site, and so on)?

Just trying to understand what you need :)

**Comment by colomet** [comment]
My situation.

I have a team. The team works in different books. Some times in a book, some times with several books at once. I need to know for each one of the members of the theam what are they doing. 

Is complicate to follow an author work.  Some times I have to open several books searching the authors work. Understanding the workflow of a selected time range.

**Comment by guillaumemolter** [comment]
@bonny I think there is two "features" related to multi-site:
- Having a central view, in the network admin, regrouping all data, from all sites would be a nice addition.
- Tracking events at the network level ( add/edit/delete sites, users, plugins and themes, network settings).

As @khromov suggested, this will require an architecture change to only use a set of central tables at the top level, vs duplicated tables for each site ( default behavior).

**Comment by jonathan-dejong** [comment]
I'll just chime in here and say that I agree. This is a much needed feature for multisite installations that would be so nice to have! :) 

A simple example scenario:
Someone has deactivated a plugin, we need to figure out when it was deactivated and who did it. The plugin was network activated and thus, no history is found. The end.

**Comment by PauloBoaventura** [comment]
A suggestion...

Is it possible to enter only one website URL for integration with another website?

Example;

Master >>
>> add WP 01 === url 01.com
>> add 02 = url002.it

Thus, the Master identifies the sites that have the plugin.

As an API or read-only authorization.


It would be possible?

**Comment by bonny** [comment]
> A suggestion...
> 
> Is it possible to enter only one website URL for integration with another website?
> 
> Example;
> 
> Master >>
> 
> > > add WP 01 === url 01.com
> > > add 02 = url002.it
> 
> Thus, the Master identifies the sites that have the plugin.
> 
> As an API or read-only authorization.
> 
> It would be possible?

Sorry but not sure I follow what you are suggesting or what you want to achieve.

**Comment by fvdm** [comment]
@bonny These two features would be great to have. Our network contains over 10 sites. It's a pain to go through each history log manually all the time.

> - Having a central view, in the network admin, regrouping all data, from all sites would be a nice addition.
> 
> - Tracking events at the network level ( add/edit/delete sites, users, plugins and themes, network settings).

**Comment by PauloBoaventura** [comment]
I've been thinking about creating an interface...
with the information from the logs of each database...
So it doesn't get too heavy... I think Wordpress development (overloads processing so many logs and queries to multiple databases)

In processing, in my view...

A CRON would import immediately so as not to overload the sites...

Would implementing this admin interface in WordPress be too complicated? In your opinion?

  Due to the large number of logs...
Basically, we must have a history of at most 1 month.

Indicate for study an administrative "drashboard" of websites in wordpress?

By analyzing them, we can start a project...

Sorry for my English...

**Comment by bonny** [comment]
@PauloBoaventura I think it would be doable. Give it a try and head over to https://wordpress.org/support/plugin/simple-history/ if you have questions or need input from other users.
